guides/source/command_line.md

```diff
@@ -101,7 +101,6 @@ If you wish to skip some files or components from being generated, you can appen
 | `--skip-active-record`  | Skip Active Record files                                    |
 | `--skip-active-storage` | Skip Active Storage files                                   |
 | `--skip-action-cable`   | Skip Action Cable files                                     |
-| `--skip-sprockets`      | Skip Sprockets files                                        |
```

Comment:
`--skip-sprockets` has been replaced with `--skip-asset-pipeline` (see https://github.com/rails/rails/commit/fb1ab3460a676ce7def0819c2e92289ef2dcbe3b#diff-a83245368df35c610a37260d0a51041de89211857d47cfd0b2d1301a11709164R61).  Would you mind adding `--skip-asset-pipeline` here?

Reply:
Sure, I will add it.

<img width="1056" alt="Screenshot 2022-02-12 at 10 22 06 AM" src="https://user-images.githubusercontent.com/56545288/153697209-cc920800-4692-41b4-b510-ab9bed2d8516.png">

It needs to be changed here as well right?

Reply:
@jonathanhefner Done!

Reply:
Yes, wherever `--skip-sprockets` is mentioned.  (The surrounding text might also need adjusting to reflect that Sprockets isn't the only possible choice for asset pipeline.)

Reply:
Looks like there's one more occurrence in `guides/source/command_line.md` on line 447: https://github.com/rails/rails/blame/97d5716f3733a91bfa9cebd3bbaca294ea81ce2a/guides/source/asset_pipeline.md#L447